<launch>
    <node pkg = "planning" type = "waypoint_server" name = "waypoints_server" output = "screen"/>
    <arg name="node_start_delay" default="5.0" />  
    <node pkg = "planning" type = "global_path_planner" name = "global_path_planner" output = "screen" launch-prefix="bash -c 'sleep $(arg node_start_delay); $0 $@' " />
    
    <!-- include other launch file -->
    <include file="$(find planning)/param/move_base.xml" />
</launch>

